README.rst

```diff
@@ -1,3 +1,18 @@
+.. image:: https://img.shields.io/pypi/v/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+
+.. image:: https://img.shields.io/pypi/l/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+
+.. image:: https://img.shields.io/pypi/wheel/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+
+.. image:: https://img.shields.io/pypi/pyversions/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+
+.. image:: https://travis-ci.org/Pylons/waitress.svg?branch=master
+    :target: https://travis-ci.org/Pylons/waitress
+
```

Comment:
Waitress does not have release branches at this time, and it's unlikely that it ever will. I was striving for consistency with Pyramid's badges [1.8-branch](https://github.com/pylons/pyramid/tree/1.8-branch#pyramid) and [master](https://github.com/pylons/pyramid/tree/master#pyramid) (from master we will soon cut a 1.9 release), where we do have branches and support at least two stable branches, so the PyPI badges don't work for Pyramid. I don't know whether consistency across Pylons Project projects is wanted for all badges (because that implies no PyPI badges), but I wanted to point it out.

In any case, I would like to have at least those three badges: docs, build status, and IRC. Would you please add docs and IRC badges to your PR?

```
.. image:: https://readthedocs.org/projects/waitress/badge/?version=latest
        :target: http://docs.pylonsproject.org/projects/waitress/en/latest/
        :alt: Latest Documentation Status

.. image:: https://img.shields.io/badge/irc-freenode-blue.svg
        :target: https://webchat.freenode.net/?channels=pyramid
        :alt: IRC Freenode
```

#pyramid is the de facto IRC channel for Pylons Project projects.